main.py

```diff
@@ -389,23 +398,28 @@ def analyse_tag(
 
     tag_type = release["object"]["type"]
     tag_url = release["object"]["url"]
-    tag_details = get_json_cached(tag_url)
-    if tag_type == "tag":
+    html_url = project_info["html_url"] + "/releases/tag/" + quote_plus(tag_name)
+
+    return ReleaseTag(version=version, page=html_url, release_type=release_type, tag_type=tag_type, tag_url=tag_url)
+
+def get_date_from_tagurl( release: ReleaseTag) -> Release:
```

Comment:
Please run `black .` (Black should have been installed with the dev dependencies)